lib/_http_outgoing.js

```diff
@@ -427,7 +428,7 @@ function _storeHeader(firstLine, headers) {
         (state.contLen || this.useChunkedEncodingByDefault || this.agent);
     if (shouldSendKeepAlive) {
       header += 'Connection: keep-alive\r\n';
-      if (this._keepAliveTimeout) {
+      if (this._keepAliveTimeout && this._defaultKeepAlive) {
         const timeoutSeconds = MathFloor(this._keepAliveTimeout) / 1000;
```

Comment:
This is wrong. Should be:

`MathFloor(this._keepAliveTimeout / 1000)`

@atian25 Would you mind fixing this as well?

Reply:
done

Reply:
but... wonder how could it pass the previous unittest?